### PR TITLE
Add a beta registry pre-release

### PR DESCRIPTION
**Pull request by milosgajdos**
See: https://github.com/distribution/distribution/releases/tag/v3.0.0-beta.1

### CODE REVIEW

**Review by thaJeztah — COMMENTED** [review]
LGTM

File: library/registry [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
              Sebastiaan van Stijn (@thajeztah)
-GitRepo: https://github.com/docker/distribution-library-image.git
+GitRepo: https://github.com/distribution/distribution-library-image.git
 GitFetch: refs/heads/master
 
+Tags: 3.0.0-beta.1
+Architectures: amd64, arm64v8, arm32v6, arm32v7, ppc64le, s390x
+GitCommit: a943e89c3efe06134cd9a4b439203c5341082cbc
+
 Tags: 3.0.0-alpha.1
```
Comment (tianon) [inline library/registry:10]: The release of `beta.1` means that `alpha.1` is now effectively unsupported, right? (ie, there won't ever be an `alpha.2` now that `beta.1` is out, right?)

So this section should be removed/replaced. :+1:

**Comment by tianon** [comment]
Yeah, unfortunately we do need at least GitHub handles (email address is optional) -- we use it most often for notifying maintainers of things (such as the ping on this PR, but also pings like https://github.com/docker-library/official-images/pull/16801#issuecomment-2211490133 for base image EOL notifications).  I really wish GitHub had some way to be able to specify that a "team" should be public so we could let folks list a team here instead, but it won't let us @-mention teams from a different organization (even if we're members of it!)

File: library/registry (hunk 1 above)
Comment (milosgajdos) [reply library/registry]: Ok I binned that: https://github.com/docker-library/official-images/pull/17151/commits/698473001c2ae3277d810726e0be480261ad3667

**Comment by milosgajdos** [comment]
> I really wish GitHub had some way to be able to specify that a "team" should be public so we could let folks list a team here instead, but it won't let us @-mention teams from a different organization (even if we're members of it!)

Yeah this would be the most welcome...alas 😮‍💨

**Review by tianon — APPROVED** [review]

File: library/registry (hunk 1 above)
Comment (thaJeztah) [reply library/registry]: Good catch; thank you! 🙏

**Comment by thaJeztah** [comment]
> but it won't let us @-mention teams from a different organization (even if we're members of it!)

would it be an option to create teams (with no additional permissions) in this org? So just for pinging / possibly review approval? I guess that could involve extra work to maintain though

**Comment by tianon** [comment]
Yeah, that would then be _us_ maintaining the list, which I think would be even more error prone than y'all doing so :sweat:

**Pull request merged**